src/Build/BackEnd/Components/RequestBuilder/AssemblyLoadsTracker.cs

```diff
@@ -107,19 +83,47 @@ public void Dispose()
             StopTracking();
         }
 
+        private static bool IsBuiltinType(string? typeName)
+        {
+            if (string.IsNullOrEmpty(typeName))
+            {
+                return false;
+            }
+
+            return typeName!.StartsWith("Microsoft.Build", StringComparison.OrdinalIgnoreCase) ||
+                   typeName.StartsWith("Microsoft.NET.Build", StringComparison.OrdinalIgnoreCase) ||
+                   typeName.StartsWith("Microsoft.CodeAnalysis.BuildTasks", StringComparison.OrdinalIgnoreCase);
```

Comment:
I don't think we should consider the Roslyn tasks built in. One of the biggest issues we have to investigate is why Roslyn tasks don't laod the way we expect precisely because they're not built in.

Reply:
Makes sense - removed from filter.
Just to supplement the info - in case of need it's possible to force all tracking via defining `MSBUILDLOGALLASSEMBLYLOADS` env var.